microservices-demo/release-scripts/make-docker-images.sh

```diff
@@ -37,10 +37,9 @@ while IFS= read -d $'\0' -r dir; do
     (
         cd "${builddir}"
         log "Building: ${image}"
```

Comment:
Now that you removed the docker push line, maybe you should remove `log "Pushing: ${image}"` down below. Then, change `log "Building: ${image}"` to `log "Building and pushing: ${image}"`, no ?

Reply:
Done in latest commit. Thanks